src/snowflake/snowpark/dataframe.py

```diff
@@ -5552,7 +5552,10 @@ def session(self) -> "snowflake.snowpark.Session":
 
     @publicapi
     def describe(
-        self, *cols: Union[str, List[str]], _emit_ast: bool = True
+        self,
+        *cols: Union[str, List[str]],
+        strings_include_math_stats=False,
```

Comment:
Could you please add AST support for the new parameter as well? The guide/steps are linked in the PR template, but it should be quick. I can walk you through it or just push a commit with the necessary changes too.